etc/geoserver/workspaces/MobiData-BW/styles/mdbw_sharing_stations_default.sld

```diff
@@ -1,21 +1,64 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <StyledLayerDescriptor version="1.0.0"
-  xsi:schemaLocation="http://www.opengis.net/sld http://schemas.opengis.net/sld/1.0.0/StyledLayerDescriptor.xsd"
-  xmlns="http://www.opengis.net/sld" xmlns:ogc="http://www.opengis.net/ogc"
-  xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-
+                       xsi:schemaLocation="http://www.opengis.net/sld http://schemas.opengis.net/sld/1.0.0/StyledLayerDescriptor.xsd"
+                       xmlns="http://www.opengis.net/sld" xmlns:ogc="http://www.opengis.net/ogc"
+                       xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <NamedLayer>
     <Name>mdbw_sharing_stations_default</Name>
     <UserStyle>
-      <Title>MobiData-BW Sharing-Stationen</Title>
+      <Title>MobiData BW® - Sharing-Stationen</Title>
       <FeatureTypeStyle>
+        <Rule>
+          <Title>Keine Echtzeitdaten</Title>
+          <ogc:Filter>
+            <ogc:Or>
+              <ogc:PropertyIsLike wildCard="*" singleChar="#" escape="!">
+                <ogc:PropertyName>feed_id</ogc:PropertyName>
+                <ogc:Literal>stadtmobil*</ogc:Literal>
+              </ogc:PropertyIsLike>
+              <ogc:PropertyIsEqualTo>
+                <ogc:PropertyName>feed_id</ogc:PropertyName>
+                <ogc:Literal>my-e-car</ogc:Literal>
+              </ogc:PropertyIsEqualTo>
+            </ogc:Or>
+          </ogc:Filter>
+          <PointSymbolizer>
+            <Graphic>
+              <Mark>
+                <WellKnownName>circle</WellKnownName>
+                <Fill>
+                  <CssParameter name="fill">#615fdf</CssParameter>
+                </Fill>
+                <Stroke>
+                  <CssParameter name="stroke">#000000</CssParameter>
+                  <CssParameter name="stroke-width">0.5</CssParameter>
+                </Stroke>
+              </Mark>
+              <Size>10</Size>
+            </Graphic>
+          </PointSymbolizer>
+        </Rule>
         <Rule>
           <Title>Fahrzeuge verfügbar</Title>
           <ogc:Filter>
-            <ogc:PropertyIsGreaterThan>
-              <ogc:PropertyName>num_vehicles_available</ogc:PropertyName>
-              <ogc:Literal>0</ogc:Literal>
-            </ogc:PropertyIsGreaterThan>
+            <ogc:And>
+              <ogc:Not>
+                <ogc:Or>
+                  <ogc:PropertyIsLike wildCard="*" singleChar="#" escape="!">
+                    <ogc:PropertyName>feed_id</ogc:PropertyName>
+                    <ogc:Literal>stadtmobil*</ogc:Literal>
+                  </ogc:PropertyIsLike>
+                  <ogc:PropertyIsEqualTo>

```

Comment:
Shouldn't flinkster be listed here as well?

Reply:
According to @ThorstenFroehlinghaus, Flinkster has realtime data.

Reply:
For now, I won't include this feed until it was clarified if it is realtime or not.